src/Compilers/CSharp/Portable/Binder/SwitchBinder.cs

```diff
@@ -14,9 +14,11 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class SwitchBinder : LocalScopeBinder
     {
-        private readonly SwitchStatementSyntax _switchSyntax;
-        private TypeSymbol _switchGoverningType;
-        private readonly GeneratedLabelSymbol _breakLabel;
+        protected readonly SwitchStatementSyntax _switchSyntax;
+        protected readonly GeneratedLabelSymbol _breakLabel;
```

Comment:
Will change `_switchSyntax` to `SwitchSyntax`. `_breakLabel` will become `private`.

---
In reply to: [70175221](https://github.com/dotnet/roslyn/pull/12416#discussion_r70175221) [](ancestors = 70175221)